src/main/java/org/cloudfoundry/promregator/config/TargetAuthenticatorConfiguration.kt

```diff
@@ -0,0 +1 @@
+package org.cloudfoundry.promregator.config
```

Comment:
Missing here intentionally any class?
Why not get rid of the file as a whole?

Reply:
Good catch. I'll delete this file